analysis/semsim/unifiedmodel.py

```diff
@@ -0,0 +1,84 @@
+from transformers import AutoTokenizer
+import datasets
+
+
+class RRUMDatasetArrow():
+    tokenizer = AutoTokenizer.from_pretrained(
+        'cross-encoder/stsb-roberta-base')
+    _scalar_features = ['channel_sim']
```

Comment:
Move scalar_features to init too?

Reply:
This is intended to express that _scalar_features will not vary between instantiations, but I don't feel too strongly about it.  Your thoughts?

Reply:
Oh yea with `_text_types` and `_text_features` there is possible variation based `with_transcript` value on instantiation. So this is cool as it is 👍